keras/src/layers/layer.py

```diff
@@ -1638,11 +1640,11 @@ def compute_size(x):
                 not_rematted_activation = self.activation
                 try:
                     self.activation = remat.remat(not_rematted_activation)
-                    return layer_call(*args, **kwargs)
+                    return layer_call
```

Comment:
Oh... now this one won't work anymore. If you don't call here, the activation gets reverted and by the time it's called, it's not rematerialized anymore.